### DISCUSSION TITLE
Sebagian Plot Hasil Visualisasi Tidak Muncul Pada Dashboard Falsk UI- Capstone Project

### DISCUSSION THREAD

**Discussion by FrengkyLR** [teamalgoritma/community#41] [Q&A] [answered]
Halo kak mau nanya, kenapa pada dashboard yang saya buat menggunakan Flask ada sebagian hasil visualisasi plot yang tidak keluar ya ?
dibawah ini hasil dashboard yang saya buat :
![flask error document](https://user-images.githubusercontent.com/101881998/162378333-dbcdad2d-b6a0-473f-b844-5362636356de.PNG)
nah untuk keterangan error pada visual studio code seperti ini :
![flaskerror](https://user-images.githubusercontent.com/101881998/162378412-2839e49c-3c46-4ed1-b6b2-524c21f42778.PNG)
Minta solusinya ya kak

**Comment by Litaa** [comment]
Halo Pak Frengky,

Warning yg terjadi di atas kemungkinan disebabkan oleh versi matplotlib yang tidak support pada tampilan visualisasi. Silahkan boleh dicek kembali menggunakan perintah `pip list` pada virtual environmentnya, apabila versi matplotlib masih 3.5.0, Bapak dapat melakukan downgrade menjadi versi 3.3.3 (sesuai dengan versi yg terdapat pada requirements.txt) dengan perintah : pip install matplotlib==3.3.3
Dan Bapak juga dapat memastikan kembali bahwa file index.html sudah terisi ya pak :) 

Terima kasih. Semoga membantu menjawab

**Reply by FrengkyLR** [reply]
semua versi package sudah saya sesuaikan kak, tetapi hasilnya tetap sebagian hasil plot tidak muncul di dashboard,  sebagian bisa muncu
l

**Comment by amaliasf** [comment, marked as answer]
Halo Pak Frengky, silahkan cek kembali pada file **index.html** di bagian **img src**. Pastikan untuk path image yang ditampilkan sudah sesuai. (Untuk penjelasan lebih detail, silahkan refer pada video '9. Cara kerja index html'). Semoga membantu Pak :)

**Reply by FrengkyLR** [reply]
Oh iya saya baru tau ternyata ada eksekusi lagi pada index.html nya, akhirnya sudah solved dan visualisasi dashboard sudah lengkap, Thanks Kak !
